packages/wrangler/src/__tests__/helpers/mock-upload-worker.ts

```diff
@@ -1,7 +1,13 @@
 import { http, HttpResponse } from "msw";
 import { ParseError } from "../../parse";
-import { getSubdomainValues } from "../../triggers/deploy";
-import { mockGetWorkerSubdomain } from "./mock-workers-subdomain";
+import {
+	getSubdomainValues,
+	getSubdomainValuesAPIMock,
```

Comment:
I was deeply confused by this being called a Mock while existing in the production code. Now I read through I see that we are simulating the backend logic within Wrangler so that we can do diffing, right?
I see there is a comment in there that we should be able to get this from the API. Is that on the roadmap?